firmware/examples/GestureSensor/source/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include <project_config.hpp>
+
+#include <cstdint>
+
+#include "L1_Drivers/i2c.hpp"
+#include "L2_Utilities/log.hpp"
+#include "L2_Utilities/time.hpp"
+#include "L3_HAL/apds_9960.hpp"
+
+int main(void)
+{
+    Apds9960 sensor;
+
+    printf("Starting Apds9960 Example\n");
+    printf("Initializing Gesture Sensor...\n");
+    if (sensor.FindDevice())
+    {
+        sensor.Initialize();
+        sensor.EnableGesture();
+    }
+    else
+    {
+        printf("Couldn't initialize sensor");
+        while (1)
+        {
+            // do nothing
+        }
+    }
+
+    Apds9960::Gesture value;
+    printf("Start collecting gestures:\n");
+
+    while (1)
+    {
+        value = sensor.GetGesture();
+        if (value != Apds9960::Gesture::kError)
+        {
+            printf("%c\n", value);
```

Comment:
Should probably print more then the character. Like `printf("Detected gesture to the RIGHT");`